test/checkpoint/checkpointer.c

```diff
@@ -212,34 +165,42 @@ real_checkpointer(void)
     WT_RAND_STATE rnd;
     WT_SESSION *session;
     wt_timestamp_t stable_ts, oldest_ts, verify_ts;
-    uint64_t delay;
+    uint32_t delay;
     int ret;
-    char buf[128], timestamp_buf[64];
-    const char *checkpoint_config;
+    char buf[128], flush_tier_config[128], timestamp_buf[64];
+    const char *checkpoint_config, *ts_config;
+    bool flush_tier;
 
-    checkpoint_config = "use_timestamp=false";
+    ts_config = "use_timestamp=false";
     verify_ts = WT_TS_NONE;
+    flush_tier = false;
 
-    if (g.running == 0)
+    if (!g.opts.running)
         return (log_print_err("Checkpoint thread started stopped\n", EINVAL, 1));
 
     __wt_random_init(&rnd);
-    while (g.ntables > g.ntables_created && g.running)
+    while (g.ntables > g.ntables_created && g.opts.running)
         __wt_yield();
 
     if ((ret = g.conn->open_session(g.conn, NULL, NULL, &session)) != 0)
         return (log_print_err("conn.open_session", ret, 1));
 
     if (g.use_timestamps)
-        checkpoint_config = "use_timestamp=true";
+        ts_config = "use_timestamp=true";
 
     if (!WT_PREFIX_MATCH(g.checkpoint_name, "WiredTigerCheckpoint")) {
-        testutil_check(
-          __wt_snprintf(buf, sizeof(buf), "name=%s,%s", g.checkpoint_name, checkpoint_config));
+        testutil_check(__wt_snprintf(buf, sizeof(buf), "name=%s,%s", g.checkpoint_name, ts_config));
         checkpoint_config = buf;
-    }
+    } else
+        checkpoint_config = ts_config;
+
+    testutil_check(__wt_snprintf(
+      flush_tier_config, sizeof(flush_tier_config), "flush_tier=(enabled,force),%s", ts_config));
+
+    /* Set up a random delay up to 5000 microseconds for the next flush. */
+    g.opts.tiered_flush_interval_us = __wt_random(&rnd) % 5001;
```

Comment:
There are two places we do this initialization. Maybe factor it into a helper function/macro so there's just one place to update if we ever want to change the delay value.

Reply:
Yeah, I agree with @keitharnoldsmith here. Calling a helper function to set the time interval would be more readable.

Reply:
Done.